Research/Research/AnswerTypeObjects.swift

```diff
@@ -39,7 +39,7 @@ public final class AnswerTypeSerializer : AbstractPolymorphicSerializer, Polymor
         """
         `AnswerType` is used to allow carrying additional information about the properties of a
         JSON-encoded `AnswerResult`.
-        """
+        """.replacingOccurrences(of: "\n", with: " ").replacingOccurrences(of: "  ", with: "\n")
```

Comment:
What's up with these string replacements? It's not obvious to me what problem this is solving.

Reply:
Formatting of the string in the JSON. It looks really weird with the newlines.